### ISSUE TITLE
Links to CarpentryCon2018 materials from website are broken

### ISSUE THREAD

**Issue by npch** [carpentries/carpentrycon#164]
A repository rearrangement has meant that the links from the CarpentryCon2018 website to the materials from the sessions are broken.

These links are from this page: https://2018.carpentrycon.org/

As an example, the Skill-Up session 1 currently links to:
https://github.com/carpentries/carpentrycon/blob/master/Sessions/2018-05-30/01-Skill-Up-1-Project-Leadership/Abstract.md

But the material has moved to:
https://github.com/carpentries/carpentrycon/blob/master/CarpentryCon-2018/Sessions/2018-05-30/01-Skill-Up-1-Project-Leadership/Abstract.md

(an extra "CarpentryCon-2018" has been added to the path)

**Comment by ChristinaLK** [comment]
Good catch @npch. I'll bring this to the attention of our website group.
